cli/pcluster/config_sanity.py

```diff
@@ -38,9 +38,15 @@ def __init__(self, region, aws_access_key_id, aws_secret_access_key):
         :param aws_secret_access_key: AWS secret access key
         """
         self.region = region
+        self.sts_endpoint_url = "https://sts." + region + "." + self.__get_sts_domain()
```

Comment:
we could define a `__get_sts_endpoint_url()` to create the whole url:
```py
self.sts_endpoint_url = self.__get_sts_endpoint_url()
```

and I'd use `format` rather than a list of append:
```py
sts_endpoint_url = "https://sts.{0}.{1}".format(self.region, "amazonaws.com.cn" if self.region.startswith("cn-") else "amazonaws.com")
```